server/src/webserver.rs

```diff
@@ -255,6 +255,140 @@ async fn save_source(
     save_source_response(true, "")
 }
 
+    //#[put("/fs/{path:.*}")]
+    // <h3>Testing input to save file</h3>
+    // <p>This function is only used for testing; it does not actively save
```

Comment:
Change/fix -- this should be called by the `save_source` function, so the production code is tested.

Reply:
Done!